tsl/src/nodes/decompress_chunk/decompress_chunk.h

```diff
@@ -63,6 +63,7 @@ typedef struct DecompressChunkPath
 	List *compressed_pathkeys;
 	bool needs_sequence_num;
 	bool reverse;
+	bool batch_merge_append;
```

Comment:
Let's rename it to "sorted merge" everywhere?

Reply:
I renamed this globally to "sorted merge"